static/src/javascripts/boot.js

```diff
@@ -58,13 +57,7 @@ const go = () => {
 
             if (!recordedConsentTime) {
                 recordedConsentTime = true;
-                cmp.willShowPrivacyMessage().then(willShow => {
-                    trackPerformance(
-                        'consent',
-                        'acquired',
-                        willShow ? 'new' : 'existing'
-                    );
-                });
+                cmp.willShowPrivacyMessage();
```

Comment:
This probably is not needed anymore as it was used to determine the willShow param. 

willShowPrivacyMessage returns promise that resolves to `true` if the CMP will show the initial privacy message once it has initialised, or `false` if not.

Reply:
I just want to be clear on this. Do you mean the whole conditional statement or just line 60?

Reply:
Sorry, just line 60.

Reply:
It's ok I just wanted to be double check before I remove anything. Addressed in 672b13705366f884f084f722cce7888b504e79ec

Reply:
@mxdvl Also this conv. happened and I asked if I should remove the whole condition but told to remove just line 60.